doc/classes/ProjectSettings.xml

```diff
@@ -2861,6 +2861,9 @@
 			[b]Note:[/b] Some platforms may restrict the actual value.
 		</member>
 		<member name="rendering/rendering_device/vulkan/max_descriptors_per_pool" type="int" setter="" getter="" default="64">
+			The number of descriptors per pool. Godot's Vulkan backend uses linear pools for descriptors that will be created and destroyed within a single frame. Instead of destroying every single descriptor every frame, they all can be destroyed at once by resetting the pool they belong to.
+			A larger number is more efficient up to a limit, after that it will only waste RAM (maximum efficiency is achieved when there is no more than 1 pool per frame). A small number could end up with one pool per descriptor, which negatively impacts performance.
```

Comment:
```suggestion
			A larger number is more efficient up to a limit, after which it will only waste RAM (maximum efficiency is achieved when there is no more than 1 pool per frame). A small number could end up with one pool per descriptor, which negatively impacts performance.
```
Is there some guideline for actual numbers that are reasonable here, besides the default of 64? Does it depend on hardware?

Reply:
Hi, it's not HW dependent. The value of 64 is arbitrary.

Ideally this should not be necessary, see my [main post](https://github.com/godotengine/godot/pull/99257#issue-2660367614) when I refer to it:

> **Possible improvement for later:** It should be possible for Godot to adapt to how many descriptors per pool are needed on a per-key basis (i.e. grow their capacity like `std::vector` does) after rendering a few frames; which would be better than the current solution of having a single global value for all pools (`max_descriptors_per_pool`) that the user needs to tweak.

Reply:
Seems fine as-is then, if the default value is reasonable and the existing description describes the tradeoffs